addons/hr/models/hr_employee.py

```diff
@@ -347,16 +347,21 @@ def _get_user_creation_notification_action(message, message_type, next_action):
                     }
                 }
 
+        existing_users_email = [record['login'] for record in self.env['res.users'].search_read([], ['login'])]
```

Comment:
Why not search users with `('login', 'in', self.mapped('name'))`? Seems more efficient.
Also could have been `.search([]).mapped('login')` I think (and or could maybe use a set)